src/App.js

```diff
@@ -1,15 +1,30 @@
+// eslint-disable-next-line
 import { Component } from 'react';
 // eslint-disable-next-line
 import DisplayTodo from './components/DisplayTodo';
 // eslint-disable-next-line
 import AddToDO from './components/AddToDo';
+// eslint-disable-next-line
+import Header from './components/Header';
+// eslint-disable-next-line
+import NavBar from './components/NavBar';
 
 export default class App extends Component {
   constructor(props) {
     super(props);
     this.state = { todo: [] };
   }
 
+  componentDidMount() {
+    if (localStorage.getItem('todos') === null) {
+      localStorage.setItem('todos', JSON.stringify(this.state.todo));
+    } else {
+      this.setState({
+        todo: JSON.parse(localStorage.getItem('todos')),
+      });
+    }
+  }
+
   onHandleAddToDO = (content) => {
     const todoObj = { content: null, complete: false, id: null };
     const todo = JSON.parse(localStorage.getItem('todos'));
```

Comment:
Here you can add the content which is passed as a parameter to the object directly like this: 
` const todoObj = { content: content, complete: false, id: null };`, then proceed to remove line 31.